src/Chainweb/Pact/RestAPI/Server.hs

```diff
@@ -361,12 +360,12 @@ spvHandler l cutR cid chainR (SpvRequest rk (Pact.ChainId ptid)) = do
     bdb = _chainResBlockHeaderDb chainR
     pdb = view CutDB.cutDbPayloadCas cdb
     b64 = TransactionOutputProofB64
-      . LBS.toStrict
+      . decodeUtf8
+      . BSL8.toStrict
       . Base64.encode
```

Comment:
Everywhere else in chain web we use unpadded base64. Shouldn't we be consistent throughout the chainweb API?

Reply:
In `Chainweb.Utils` we already have `encodeB64UrlText` (or `encodeB64UrlTextNoPadding`, if we want to be consistent).

Reply:
In Pact we're using unpadded Base64 so yes, definitely we should standardize on this.

Reply:
But this code is using padded Base64.

Reply:
Switched to yours 😄